README.md

```diff
@@ -41,11 +41,134 @@ Running your app using IPFS will mimic the production environment that will be u
 - **build:app**: Builds the front-end
 - **build:script**: Builds the background script
 - **test**: Runs tests for the contracts
-- **publish:minor**: Release a minor version to aragonPM
+- **publish:patch**: Release a patch version of the frontend to aragonPM
+- **publish:minor**: Release a minor version of the frontend to aragonPM
```

Comment:
For the above, I'd reword as "Release a patch/minor version to aragonPM (only frontend content changes allowed)"

Reply:
Please see https://github.com/aragon/aragon-cli/issues/310 I'm trying to figure out how major/minor/patch work, based on the aragonPM restrictions. Shouldn't it be possible to release a bugfix to the contract for example?